lib/pdk/module/build.rb

```diff
@@ -120,8 +120,14 @@ def stage_path(path)
         elsif File.symlink?(path)
           warn_symlink(path)
         else
+          validate_ustar_path!(relative_path.to_path)
           FileUtils.cp(path, dest_path, preserve: true)
         end
+      rescue ArgumentError => e
+        raise PDK::CLI::ExitWithError, _(
+          '%{message} Please rename the file or exclude it from the package ' \
+          'by adding it to the .pdkignore file in your module.',
```

Comment:
I think the subtle distinction between the two failure cases is probably not worth surfacing to the user in this message and we can just say that the path is too long here. We can log the distinction at DEBUG perhaps?

Reply:
I was thinking that, but say there is a file in the root of the module called `this_is_a_file_name_that_goes_over_100_characters_oh_no.md`, it would fail but would clearly be < 256 bytes which would make a generic error confusing.